src/github/entities/user-installation.entity.ts

```diff
@@ -0,0 +1,15 @@
+import { Base } from 'src/universal/base.entity';
+import { Column, Entity, ManyToOne } from 'typeorm';
+import { User } from './user.entity';
+
+@Entity()
+export class UserInstallation extends Base {
+  @Column({ type: 'varchar', length: 20 })
+  installationId: string;
+
+  @Column({ type: 'varchar', length: 255 })
+  organizationName: string;
+
+  @ManyToOne(() => User, (user) => user.installations)
+  user: User;
+}
```

Comment:
PullProofAI

> Note: This review was outside of the patch, but no patch was found that overlapped with it. Original lines [23-23]

Consider adding validation for `installationId` and `organizationName` properties to ensure data integrity.
```diff
+   @Column({ type: 'varchar', length: 20, nullable: false })
+   installationId: string;
+
+   @Column({ type: 'varchar', length: 255, nullable: false })
+   organizationName: string;
```


<!-- This is an auto-generated comment by PullProofAI -->